pkgs/applications/science/math/cemu/ltcg.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/gui/qt/CEmu.pro b/gui/qt/CEmu.pro
+index 1e509d12..107df284 100644
+--- a/gui/qt/CEmu.pro
++++ b/gui/qt/CEmu.pro
+@@ -52,7 +52,7 @@ TEMPLATE = app
+ # Localization
+ TRANSLATIONS += i18n/fr_FR.ts i18n/es_ES.ts i18n/nl_NL.ts
+ 
+-CONFIG += c++11 console
++CONFIG += c++11 console ltcg
```

Comment:
This patch could be avoided by adding `"CONFIG+=ltcg"` in `qmakeFlags`

Reply:
Neat! Thanks